test/module/irohad/consensus/yac/mock_yac_crypto_provider.hpp

```diff
@@ -27,7 +27,7 @@ namespace iroha {
         auto tmp =
             shared_model::crypto::DefaultCryptoAlgorithmType::generateKeypair()
                 .publicKey();
-        std::string key(tmp.blob().size(), 0);
+        std::string key(tmp.blob().size(), '0');
```

Comment:
Please confirm that the char '0' is expected here.

Reply:
confirm